packages/locales/src/nl.xlf

```diff
@@ -62,6 +62,10 @@
   <source>Please fill in this field.</source>
   <target>Voer een waarde in.</target>
 </trans-unit>
+<trans-unit id="sde6884478b7cc818">
+  <source>Please enter at least <x id="0" equiv-text="${this.minLength}"/> characters (you currently have <x id="1" equiv-text="${length}"/> character<x id="2" equiv-text="${length &gt; 1 ? 's' : ''}"/>).</source>
+  <target>Voer tenminste <x id="0" equiv-text="${this.minLength}"/> karakters in (je hebt op dit momoment <x id="1" equiv-text="${length}"/> karakter<x id="2" equiv-text="${length &gt; 1 ? 's' : ''}"/>).</target>
```

Comment:
I think "tekens" is easier to understand than "karakters"

Reply:
Let's look at all the translations together later :)